third_party/bitcoin-core/BUILD.gn

```diff
@@ -36,12 +36,14 @@ static_library("bitcoin-core") {
   sources = [
     "src/src/base58.cpp",
     "src/src/base58.h",
+    "src/src/bech32.cpp",
+    "src/src/bech32.h",
     "src/src/crypto/ripemd160.cpp",
     "src/src/crypto/ripemd160.h",
+    "src/src/util/strencodings.h",
   ]
```

Comment:
Adding bech32 strencodings (bech32::Encode and ConvertBits) for `HDKey::GetSegwitAddress` needs